R/tables.R

```diff
@@ -188,20 +174,33 @@ format_keep_na <- function(x, ...) {
 #' @rdname postgres-tables
 #' @export
 setMethod("dbAppendTable", c("PqConnection"),
-  function(conn, name, value, ..., row.names = NULL) {
+  function(conn, name, value, copy = TRUE, warn = TRUE, ..., row.names = NULL) {
```

Comment:
We don't need a `warn` argument here, `dbAppendTable()` is a new generic.

```suggestion
  function(conn, name, value, ..., copy = TRUE, row.names = NULL) {
```